armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -427,6 +449,54 @@ def _getLinkedComponents(self, b, c):
             )
 
 
+def _determineLinked(componentA, componentB):
+    """determine axial component linkage for two components
+
+    Parameters
+    ----------
+    componentA : :py:class:`Component <armi.reactor.components.component.Component>`
+        component of interest
+    componentB : :py:class:`Component <armi.reactor.components.component.Component>`
+        component to compare and see if is linked to componentA
+
+    Notes
+    -----
+    - Requires that shapes have the getBoundingCircleInnerDiameter and getBoundingCircleOuterDiameter defined
+    - For axial linkage to be True, components MUST be solids, the same Component Class, multiplicity, and meet inner
+      and outer diameter requirements.
+
+    Returns
+    -------
+    linked : bool
+        status is componentA and componentB are axially linked to one another
+    """
+    if (
+        (componentA.containsSolidMaterial() and componentB.containsSolidMaterial())
+        and isinstance(componentA, type(componentB))
+        and (componentA.getDimension("mult") == componentB.getDimension("mult"))
+    ):
+        idA, odA = (
+            componentA.getBoundingCircleInnerDiameter(),
```

Comment:
looking at the way this is implemented it needs to be how you have structured it so we should probably rename to 
getCircleInnerDiameter